### PR TITLE
DOC-10329 - Add Capella support and example to Ottoman page

### PR DESCRIPTION
**Pull request by maria-robobug**
https://issues.couchbase.com/browse/DOC-10329

- Adds compatibility information for Ottoman and Capella
- Adds an Ottoman Capella example (pretty much the same as the existing one, but with Capella connection details instead).
- Small driveby fix for release notes in nav

### CODE REVIEW

File: modules/hello-world/pages/start-using-ottoman.adoc [hunk 1]
```diff
@@ -109,8 +116,40 @@ async function main() {
...
+. Click on the relevant cluster.
+. Select the "Connect" tab and scroll down to the "Security Certificate" section.
+. Download the "Root Certificate".
+
+[source,javascript]
+----
+include::hello-world:example$start-using-ottoman-cloud.js[tag=connect]
+----
+
+TIP: If your application is running in a xref:ref:client-settings.adoc[constrained network environment] (i.e., your laptop), you may experience timeout issues.
```
Comment (maria-robobug) [inline modules/hello-world/pages/start-using-ottoman.adoc]: Just trying to help users with this, after the confusion on slack:
https://couchbase.slack.com/archives/CC6GX1CJJ/p1661970119229539

@RichardSmedley , let me know if this seems correct for Capella, or if needs tweaking.

File: modules/hello-world/examples/start-using-ottoman-cloud.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
+'use strict'
```
Comment (maria-robobug) [inline modules/hello-world/examples/start-using-ottoman-cloud.js:1]: @ejscribner shout if any of this looks incorrect! 
Thanks for helping me out, I managed to get this working on my local against Capella with a real certificate.

**Review by ejscribner — COMMENTED** [review]
Looks good to me! Glad you were able to get it working with the cert!! 🚀

File: modules/hello-world/pages/start-using-ottoman.adoc [hunk 3]
```diff
@@ -109,8 +116,40 @@ async function main() {
...
+. Click on the relevant cluster.
+. Select the "Connect" tab and scroll down to the "Security Certificate" section.
+. Download the "Root Certificate".
+
+[source,javascript]
+----
+include::hello-world:example$start-using-ottoman-cloud.js[tag=connect]
+----
+
+TIP: If your application is running in a xref:ref:client-settings.adoc#constrained-network-environments[constrained network environment] (i.e., your laptop), you may experience timeout issues.
```
Comment (RichardSmedley) [inline modules/hello-world/pages/start-using-ottoman.adoc]: Change the i.e. to e.g., as there are other likely places.

Comment (RichardSmedley) [reply modules/hello-world/pages/start-using-ottoman.adoc]: (better yet, change it to "such as your laptop)

File: modules/hello-world/pages/start-using-ottoman.adoc [hunk 4]
```diff
@@ -109,8 +116,40 @@ async function main() {
...
+To mitigate this, you can tweak your connection settings to account for any latency or bandwidth issues that may arise.
+
+Couchbase Capella uses xref:cloud:organizations:organization-projects-overview.adoc[Roles] to control user access to database resources.
+For the purposes of this example, we can use the _Organization Owner_ role automatically assigned to a user account during installation of the Capella cluster.
+This role grants full access to a Capella organization, including full data access to all projects and clusters.
+In a production scenario, we strongly recommend setting up users with more granular access roles as a best practice.
```
Comment (RichardSmedley) [inline modules/hello-world/pages/start-using-ottoman.adoc]: Worth putting _strongly recommend_ in italics.or possibly some JS to get it to jump off the screen and tap people on the shoulder (less violent than slapping them round the face, but the minimum to make sure everyone gets the point ;) )

**Review by RichardSmedley — APPROVED** [review]
LGTM  :+1:

File: modules/hello-world/pages/start-using-ottoman.adoc (hunk 4 above)
Comment (maria-robobug) [reply modules/hello-world/pages/start-using-ottoman.adoc]: Hahaha! Brings a whole new meaning to interactive docs 🤣

**Pull request merged**